tutorials/3d/portals/appendix.rst

```diff
@@ -0,0 +1,87 @@
+
+The default portal has 4 corners as shown in the inspector:
+
+.. image:: img/portal_points.png
+
+You can edit these points in the inspector to make a better match to the opening in your game level. But bear in mind it generally better to keep the number of points as low as possible for the efficiency of the system. It is better to risk rendering a little too much than to spend the time culling objects at 20 different edges, for example.
+
+Room point editing
+~~~~~~~~~~~~~~~~~~
+You also have the option to manually edit the points used to define the convex hull of a room. These points are not present by default, you would typically create them by pressing the ``Generate Points`` button in the editor toolbar, when a room is selected. This will transfer the auto bound from the geometry (or manual ``Bound_`` mesh) into the inspector. Once there are points in the inspector, they will be used and override any other method. So if you wish to revert your manual editing, simply delete all the points.
+
+.. image:: img/room_points.png
+
+Manually editing rooms points is more difficult than editing portal points, but it can be useful in some situations, especially where the auto-bound doesn't _quite_ get the right result you want. It is usually a good idea to use a lot of ``simplification`` in the inspector for the Room before generating the points (be aware though that by default, the ``simplification`` value will be inherited from the RoomManager).
+
+RoomManager
+~~~~~~~~~~~
+
+Show Debug
+^^^^^^^^^^
+This can be used to turn on and off display of portals in the editor, and control the amount of logging. Debug will always be set to false on exported projects.
+
+Debug Sprawl
+^^^^^^^^^^^^
+This mode will only display meshes that are sprawling through portals from the current camera room. Large statics that cross portals are usually the ones you want to sprawl. Typical examples might be terrain mesh areas, or large floor or ceiling meshes. You usually don't want things like door frames to sprawl to the adjacent room - that is what fine tuning the ``Portal Margin`` is for.
+
+Merge Meshes
+^^^^^^^^^^^^
+In order to keep drawcalls to a minimum, the system offers the option to automatically merge similar ``STATIC`` meshes within a room (aka, *static batching*). This can increase performance in many cases. The culling accuracy is reduced, but as a room is a fairly logical unit for culling, this trade off usually works in your favour.
+
+Plane Simplification
+^^^^^^^^^^^^^^^^^^^^
+In some cases, automatically generated convex hull bounds may contain a very large number of planes. This is not ideal because it slows down determining which room a camera or object is within. The system can optionally simplify hulls. The degree of simplification can be selected by the user, between ``0`` (no simplification) and ``1`` (maximum simplification). You can also override this value in individual rooms.
```

Comment:
```suggestion
Plane Simplification
^^^^^^^^^^^^^^^^^^^^

In some cases, automatically generated convex hull bounds may contain a very large number of planes. This is not ideal because it slows down determining which room a camera or object is within. The system can optionally simplify hulls. The degree of simplification can be selected by the user, between ``0`` (no simplification) and ``1`` (maximum simplification). You can also override this value in individual rooms.
```